sdk/resources/Microsoft.Azure.Management.Resource/src/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Resource Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure resource management operations including Resource Groups.")]
 
-[assembly: AssemblyVersion("3.17.3.0")]
-[assembly: AssemblyFileVersion("3.17.3.0")]
+[assembly: AssemblyVersion("3.17.4.0")]
```

Comment:
If it's just a minor change, you don't need to bump `AssemblyVersion`. See: https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=signing-and-publishing-nuget-packges